atoma-proxy/src/server/handlers/chat_completions.rs

```diff
@@ -2060,9 +2065,11 @@ mod tests {
             max_completion_tokens: 10,
         };
         let tokenizer = load_tokenizer().await;
-        let result = request.get_compute_units_estimate(Some(&tokenizer));
-        assert!(result.is_ok());
-        assert_eq!(result.unwrap().max_total_compute_units, 21); // 8 tokens + 3 overhead + 10 completion
+        let result = request
+            .get_compute_units_estimate(Some(&tokenizer))
+            .unwrap();
+        assert_eq!(result.num_input_tokens, 11); // 8 tokens + 3 overhead
```

Comment:
Pardon my ignorance but the 3 tokens overhead is implicit? I wasn't aware of this, perhaps this could be a constant for better readability.

Reply:
for the overhead there is a `const MESSAGE_OVERHEAD_TOKENS: u64 = 3;`, it's not used in tests, maybe it should, I will change that